bindep.txt

```diff
@@ -1,6 +1,7 @@
-gcc [compile platform:centos-8 platform:rhel-8]
-libcurl-devel [compile platform:centos-8 platform:rhel-8]
-libxml2-devel [compile platform:centos-8 platform:rhel-8]
-openssl-devel [compile platform:centos-8 platform:rhel-8]
+gcc [compile platform:rpm]
+libcurl-devel [compile platform:rpm]
+libxml2-devel [compile platform:rpm]
+openssl-devel [compile platform:rpm]
 python38-devel [compile platform:centos-8 platform:rhel-8]
```

Comment:
Could you please change  above line to below?
```
python39-devel [compile platform:centos-8 platform:rhel-8]
```

I forgot to do that as a part of https://github.com/oVirt/ovirt-ansible-collection/pull/632

Reply:
Oops, this almost fell off my radar. Went ahead and made that change @mwperina. This should be good to go now.

Reply:
Thanks